File: Java/Armstrong.java
import java.util.Scanner;
public class Armstrong {

    public static void main (String[] args) {
        Scanner sc=new Scanner(System.in);
        int number=sc.nextInt();
        int originalNumber, remainder, result = 0, count=0;

        originalNumber = number;

        while (originalNumber != 0)
        {
            count++;
            originalNumber /= 10;
        }
        originalNumber = number;
        while (originalNumber != 0)
        {
            remainder = originalNumber % 10;
            result += Math.pow(remainder, count);
            originalNumber /= 10;
        }
        if(result == number)
            System.out.println(number + " is an Armstrong number.");
        else
            System.out.println(number + " is not an Armstrong number.");
    }
}

File: C/armstrong.c
#include < stdio.h >  
  
int main()  
{  
    int n, r, s = 0, t;  
  
    printf("Enter an integer number\n");  
    scanf("%d", &n);  
  
    t = n;  
  
    while(n)  
    {  
        r = n % 10;  
        s = s + (r * r * r);  
        n = n / 10;  
    }  
  
    if(t == s)  
    {  
        printf("%d is armstrong number\n", t);  
    }  
    else  
    {  
        printf("%d is not armstrong number\n", t);  
    }  
  
    return 0;  
}  
